examples/react/ssr-fastify-trpc/vite.config.ts

```diff
@@ -0,0 +1,16 @@
+import { dirname, join } from 'path'
+import { TanStackRouterVite } from '@tanstack/router-plugin/vite'
+import { defineConfig } from 'vite'
+import { fileURLToPath } from 'url'
+import react from '@vitejs/plugin-react'
+
+import tsrOptions from './tsr.config.json' assert { type: 'json' }
+
+// https://vitejs.dev/config/
+export default defineConfig({
+  plugins: [
+    TanStackRouterVite(tsrOptions as Parameters<typeof TanStackRouterVite>[0]),
```

Comment:
You shouldn't need to do this, the vite plugin should automatically pickup the file.

Or just simply inline the config with the file plugin here.

Reply:
i had made the change but now when i run the dev server, this plugin is not using the values i specified in the tsr.config.json file.  i can inline the configuration if you'd prefer, but since i also have a routegen script specified in package.json (which does read from tsr.config.json), i was importing that configuration here so that the route generation configuration is only specified in one place (whether the user runes the routegen script or whether the vite plugin generates the routes while the dev server is running).  perhaps i have uncovered a bug?  should the plugin look for configurations specified in the tsr.config.json file as a default as the cli does?  i will copy the configuration from the json file here for now.